src/script.js

```diff
@@ -1,18 +1,32 @@
-let myTimer;
+const long = 2;
```

Comment:
consider grouping these three into a single enum because they are a) related, and b) an exclusive list